### PR TITLE
Db set up

### PR DESCRIPTION
**Pull request by DuaneDave**
### In this PR, the database was: 

- set up
- initialised with some entries
- and specific inputs were queried

Below are the screenshots
[screenshot 1](https://github.com/DuaneDave/vet_clinic-DB/blob/DB-set-up/screenshots/one.JPG)
[screenshot 2](https://github.com/DuaneDave/vet_clinic-DB/blob/DB-set-up/screenshots/two.JPG)
[screenshot 3](https://github.com/DuaneDave/vet_clinic-DB/blob/DB-set-up/screenshots/three.JPG)
[screenshot 4](https://github.com/DuaneDave/vet_clinic-DB/blob/DB-set-up/screenshots/four.JPG)

For the wrongly done one

[UPDATE SCREENSHOT](https://github.com/DuaneDave/vet_clinic-DB/blob/DB-set-up/screenshots/five.JPG)

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT name FROM animals WHERE name LIKE '%mon';
+SELECT date_of_birth FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE neutered IS true AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+SELECT name, escape_attempts FROM animals WHERE weight_kg < 10.5;
```
Comment (gemmen29) [inline queries.sql]: - [ ] Kindly fix this query as it's not correct as per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md#:~:text=List%20name%20and%20escape%20attempts%20of%20animals%20that%20weigh%20more%20than%2010.5kg). The query should select animals with weight more than 10.5kg.

File: schema.sql [hunk 2]
```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id BIGSERIAL,
```
Comment (gemmen29) [inline schema.sql:4]: - [ ] [OPTIONAL] Kindly mark the id column as the primary key like:
```
PRIMARY KEY (id)
```

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @DuaneDave, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (DuaneDave) [reply queries.sql]: Oops my bad, I copied the code halfway.

Thanks for pointing out my shortcoming.

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @DuaneDave, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights:
- All requirements completed
- Professional readme

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**